engine/runtime/bun/worker_memory_test.ts

```
@@ -0,0 +1,564 @@
+                error: error.message,
+                stack: error.stack,
+                requestedLimit: maxMemoryMB,
+                env: process.env.COMPOZY_MAX_MEMORY_MB,
+              });
+              process.exit(1);
+            }
+          }
+        }
+      `;
+
+      eval(setupCode);
```

Biome (2.1.2)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)